manifests/profile/hathitrust/shibboleth.pp

```diff
@@ -10,13 +10,60 @@
 #   include nebula::profile::hathitrust::shibboleth
 class nebula::profile::hathitrust::shibboleth () {
   include nebula::profile::hathitrust::apache
+  include nebula::systemd::daemon_reload
 
   package {
     [
 # not in debian 9 - would need to install via https://downloads.mariadb.org/connector-odbc/
```

Comment:
I haven't done this (or verified that shib works besides appearing to start up), so this needs a new issue.